pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -55,9 +70,118 @@ func (r *ReconcileStorageCluster) ensureStorageClasses(instance *ocsv1.StorageCl
 			}
 		}
 	}
+	return nil
+}
 
-	instance.Status.StorageClassesCreated = true
-
+// ensureExternalCluster ensures that Storagclass for the external cluster exist
+func (r *ReconcileStorageCluster) ensureExternalCluster(instance *ocsv1.StorageCluster, reqLogger logr.Logger) error {
```

Comment:
If this function is supposed to ensure StorageClass for External Cluster, shouldn't it be named accordingly? `ensureExternalCluster` seems misleading.

Reply:
I see that this function also creates ConfigMaps and Secrets. So `ensureExternalClusterObjects` or `ensureExternalClusterResources` etc sounds more appropriate to me.

Reply:
Done